### PR TITLE
added partition equal subset sum

### PR DESCRIPTION
**Pull request by bhagyashree-06**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/smv1999/CompetitiveProgrammingQuestionBank/blob/master/CONTRIBUTING.md)?

yes
### Description

partition equal subset sum in python

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests

#319 
## Fill up the DCP form 

If you are a part of DCP 21, fill up this [form](https://docs.google.com/forms/d/e/1FAIpQLSeRAJnHE4eAa_W0IUeNZgheiy5POX3I3tQB7DtFsiB13D5w2g/viewform) after the PR is merged.

### The links provided in the form will be checked thoroughly, if someone tries fooling around, strict actions will be taken against the individual.

### CODE REVIEW

File: General Questions/Partition_Equal_Subset_Sum.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+arr = []
+n = len(arr)
+ 
+# Function call
+if findPartion(arr, n) == True:
+    print("True")
+else:
+    print("False")
+    
+'''
```
Comment (smv1999) [inline General Questions/Partition_Equal_Subset_Sum.py:92]: Close this multi-line comment.

**Review by smv1999 — CHANGES_REQUESTED** [review]

File: General Questions/Partition_Equal_Subset_Sum.py (hunk 1 above)
Comment (bhagyashree-06) [reply General Questions/Partition_Equal_Subset_Sum.py:92]: done.

**Review by noviicee — COMMENTED** [review]
@bhagyashree-06 Modify the code to take dynamic  input from user.

**Comment by bhagyashree-06** [comment]
@noviicee, done!
kindly check

**Comment by noviicee** [comment]
@bhagyashree-06 The entire code is a copy-paste from GfG. Even the comments!

**Comment by smv1999** [comment]
> @bhagyashree-06 The entire code is a copy-paste from GfG. Even the comments!

Well that goes against the [Contributing Guidelines](https://github.com/smv1999/CompetitiveProgrammingQuestionBank/blob/master/CONTRIBUTING.md).

**Pull request closed without merging**